test/continuous-test-suite-client.ts

```
@@ -1043,12 +1032,49 @@ async function testReactHooksExports(): Promise<boolean | null> {
-    );
+    // Hooks need React's internal dispatcher, which is only set up inside a
+    // renderer. Calling them outside any component should throw the canonical
+    // "Hook called outside of component" error from React. Catching that
+    // error proves the hook is correctly wired to React's internals; not
+    // catching it would mean the hook is a stub.
+    let invokedSignal: "passes" | "fails" = "fails";
+    let invokeFailReason = "";
+    try {
+      const useChat = (clientModule as Record<string, unknown>)
+        .useChat as () => unknown;
+      useChat();
```

Biome (2.4.15)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)